deploy/examples/csi/cephfs/pvc-restoregroup.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
```

Comment:
we already have the pvc for restore operation we can reuse that we don't need to have new yaml (almost duplicate) for this one